.gitignore

```diff
@@ -43,7 +192,7 @@ htmlcov/
 .cache
 nosetests.xml
 coverage.xml
-*.cover
+*,cover
```

Comment:
That's the gitignore generated by https://www.gitignore.io/. I don't remember what this extension is used for though, do you?

gitignore.io usually does a good job, if it's a bug we should open an issue with them.